docs/cicd.md

```diff
@@ -2,14 +2,60 @@
 
 ## Content
 
+- [Deliver a new release by triggering the CI](#triggering-a-delivery-using-the-ci)
 - [Workflows](#workflows)
+- [Services](#services)
 - [Fastlane](#fastlane)
 
+## Triggering a Delivery using the CI
+
+The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflows _triggered_native_development_delivery_ and _triggered_production_delivery_:
+
+- Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-api-tokens/).
+- Trigger a delivery by sending a POST request to:`https://circleci.com/api/v2/project/github/digitalfabrik/lunes-app/pipeline`
+  - header needs to contain `Circle-Token=your-personal-token` and `content-type=application/json`
+  - body could look like this:
+    ```json
+    {
+      "parameters": {
+        "run_commit": false,
+        "run_dev_delivery": true
+      }
+    }
+    ```
+  - If no branch is specified, main is used as default. This should normally not be changed.
+  - Per default a development delivery is made.
+  - For more information on how to use it, execute it without parameters to see usage information.
+
 ## Workflows
 
 Several workflows exist for different purposes:
 
 - **commit**: Executed for all commits of PRs to ensure good code quality and working code.
+- **dev_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers builds to development (currently only android).
+- **prod_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers builds to development (not available yet).
```

Comment:
I'd change the naming, but maybe just personal preference.

Reply:
What would you prefer?

Reply:
I will just keep it, as it makes more or less no difference.